.travis.yml

```diff
@@ -105,8 +105,9 @@ before_script:
 script:
   - |
     if [[ -v MAVEN_CHECKS ]]; then
-      ./mvnw install -DskipTests -B -T C1 -P ci -pl '!presto-server-rpm'
-      ./mvnw install -B -P ci -pl presto-server-rpm
+      ./mvnw install -DskipTests -B -T C1 -P ci -pl '!presto-server-rpm' &&
+      ./mvnw install -B -P ci -pl presto-server-rpm &&
+      true
```

Comment:
What's the purpose of `&& true` ?

Reply:
makes it unlikely we repeat the same mistake again. in this place at least

i already use it here
https://github.com/prestosql/presto/blob/d2630891726cbea623abb9aea3935e5dfceb66d6/.travis.yml#L154-L157